exporter/importables.go

```diff
@@ -1971,4 +1971,46 @@ var resourcesMap map[string]importable = map[string]importable{
 			// {Path: "http_url_spec.authorization", Variable: true},
 		},
 	},
+	"databricks_mlflow_model": {
+		Service: "mlflow-models", // why model instead of mlflow
+		Name: func(ic *importContext, d *schema.ResourceData) string {
+			return d.Id()
```

Comment:
Let's change this to `return d.Id() + "_" + d.Get("registered_model_id").(string)` because there could be different models with similar names, but having `_` instead of `-`, etc.